# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cpuid/install.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- cpuid-3.3/Makefile.vanilla	2007-06-21 17:25:27.000000000 +0200
+++ cpuid-3.3/Makefile	2007-06-21 17:26:35.000000000 +0200
@@ -1,7 +1,12 @@
+prefix=/usr/local
+
 CFLAGS=-O2 -Wall
 
 cpuid: cpuid.o
-	gcc -o $@ $^
+	$(CC) -o $@ $^
+
+install: cpuid
+	install cpuid $(DESTDIR)$(prefix)/bin
 
 clean:
 	rm -f *.o cpuid
